packages/shared/forks/ReactFeatureFlags.native-fb-dynamic.js

```diff
@@ -22,6 +22,7 @@ export const enableObjectFiber = __VARIANT__;
 export const enableHiddenSubtreeInsertionEffectCleanup = __VARIANT__;
 export const enablePersistedModeClonedFlag = __VARIANT__;
 export const enableShallowPropDiffing = __VARIANT__;
+export const enableEagerAlternateStateNodeCleanup = __VARIANT__;
```

Comment:
One thing to double check is that this `__VARIANT__` actually works for RN. I would test by adding a `throw` when the flag is true/false and confirm that some tests actually break.